test/conftest.py

```diff
@@ -942,7 +955,7 @@ def adaptive_model_qa(num_processes):
         logging.error(f"Not all the subprocesses are closed! {len(children)} are still running.")
 
 
-@pytest.fixture(scope="function")
+@pytest.fixture
 def bert_base_squad2(request):
```

Comment:
This is unrelated to this PR, but not sure about the naming of this fixture, given that the model used is not bert-base but minilm... 🤔

Reply:
OMG :smile: Let's remember about this one!